identity/identity.go

```diff
@@ -0,0 +1,64 @@
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+)
+
+func CreateNewIdentity(path string) (string, error) {
+	keystoreManager := keystore.NewKeyStore(path, keystore.StandardScryptN, keystore.StandardScryptP)
+	account, err := keystoreManager.NewAccount("")
+	if err != nil {
+		return "", err
+	}
+
+	return account.Address.Hex(), nil
+}
+
+func GetIdentities(path string) []string {
+	keystoreManager := keystore.NewKeyStore(path, keystore.StandardScryptN, keystore.StandardScryptP)
```

Comment:
`keystoreManager` is constructed several times, make it as factory `NewKeystore()` or package variable